test/options/deactivateCallback.test.ts

```diff
@@ -0,0 +1,48 @@
+import { fireEvent } from '@testing-library/dom'
+import {
+  setDocumentBody,
+  getPopover,
+  waitToStopChanging,
+  getButton,
+} from '../helper'
+import littlefoot from '../../src'
+
+test('setup with deactivateCallback', async () => {
+  setDocumentBody('single.html')
+  const deactivateCallback = jest.fn()
+  littlefoot({ deactivateCallback: deactivateCallback })
+
+  const button = getButton('1')
+
+  fireEvent.click(button)
+  await waitToStopChanging(button)
+  expect(deactivateCallback).toHaveBeenCalledTimes(0)
```

Comment:
We don't need to test for things that don't happen. This assertion would not drive any changes to the production code.

Reply:
👍